benchmarks/src/pages/MuiPage.tsx

```
@@ -0,0 +1,205 @@
+import * as React from 'react'
+import { useEffect, useMemo, useRef } from 'react'
+import {
+  LayoutList,
+  useVirtualizer,
+  type Virtualizer,
```

ESLint

[error] Member 'Virtualizer' of the import declaration should be sorted alphabetically.

(sort-imports)